### ISSUE TITLE
Other package build process failing since last commit here

### ISSUE THREAD

**Issue by SylvainSoKette** [tsconfig/bases#192]
I'm using [OpenVidu's fork of mediasoup-sdp-bridge](https://github.com/OpenVidu/mediasoup-sdp-bridge) and it started spouting errors about typescript since a few days ago.

After much pocking around, I finally understood that it came down to @tsconfig/node16 having received a commit very recently.

Here is the errror which halts the build process:
```
> mediasoup-sdp-bridge@3.6.5 typescript:build
> node npm-scripts.js typescript:build

npm-scripts.js [INFO] running task "typescript:build"
npm-scripts.js [INFO] executing command: tsc
node_modules/@tsconfig/node16/tsconfig.json:7:15 - error TS6046: Argument for '--module' option must be: 'none', 'commonjs', 'amd', 'system', 'umd', 'es6', 'es2015', 'es2020', 'esnext'.

7     "module": "Node16",
                ~~~~~~~~

Found 1 error.
```

As I understand, mediasoup-sdp-bridge is using a version of Typescript that is incompatible with tsconfig ? Or is there something wrong with tsconfig ?

**Comment by orta** [comment]
Yep, you need to use a newer version of TypeScript (or an older tsconfig base)

**Comment by SylvainSoKette** [comment]
Thanks, I'll raise the issue to the guys at OpenVidu then :)

**Comment by cruizba** [comment]
@SylvainSoKette , I've updated the typescript version at: https://github.com/OpenVidu/mediasoup-sdp-bridge

It should work now.
